test/test.bats

```diff
@@ -40,10 +40,33 @@ setup() {
 Interpreter closing...$'
 }
 
-@test "Display usage on empty args execution and say that jsonfile argument is required" {
+@test "Display global usage on empty required args" {
     run bash -c './_build/default/bin/turing | cat -e'
+    assert_output "usage: ft_turing COMMAND$
+$
+commands:$
+  serve          starts the turing machine http REST API$
+  run            runs cli turing machine command$
+$
+Run 'ft_turing COMMAND --help' for more information on a command.$"
+}
+
+@test "Display global usage on invalid required args" {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Excellent test!